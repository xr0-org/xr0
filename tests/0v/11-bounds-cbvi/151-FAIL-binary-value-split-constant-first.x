foo(int index) ~ [ setup: index = [0?2]; ]
{
	int k, arr[2];
        arr[1] = 1;
        k = arr[index]; /* arr[0] not defined */
}
