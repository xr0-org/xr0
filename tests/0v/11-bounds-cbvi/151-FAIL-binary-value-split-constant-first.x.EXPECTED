11-bounds-cbvi/151-FAIL-binary-value-split-constant-first.x:5:23: undefined memory access: *(arr+index) has no value (foo | #0 âˆˆ 0)
